src/lib.rs

```diff
@@ -5572,38 +5627,51 @@ impl TransportParams {
     /// Creates a qlog event for connection transport parameters and TLS fields
     #[cfg(feature = "qlog")]
     pub fn to_qlog(
-        &self, owner: qlog::TransportOwner, version: u32, alpn: &[u8],
-        cipher: Option<crypto::Algorithm>,
-    ) -> qlog::event::Event {
-        let ocid = qlog::HexSlice::maybe_string(
+        &self, owner: qlog::TransportOwner, cipher: Option<crypto::Algorithm>,
+    ) -> qlog::EventData {
```

Comment:
Why not return `qlog::Event` directly? Would save having to call `qlog::Event::new` from the caller.

Reply:
In the new commit, I stuck with returning EventData here so that the call site can drive the population of the event time field, which depends on access to some of the connection's parameters. Attempting to construct the event in to_qlog would require passing more parameters, which didn't seem much clearer to me.